ext/opcache/ZendAccelerator.c

```diff
@@ -1399,7 +1400,10 @@ zend_result zend_accel_invalidate(zend_string *filename, bool force)
 	realpath = accelerator_orig_zend_resolve_path(filename);
 
 	if (!realpath) {
-		return FAILURE;
+		//this can't be right - file could have been deleted, but we still need to invalidate it.
```

Comment:
"this can't be right" sounds like an internal error. But the user could also just pass something invalid to `opcache_invalidate`, no? I'd just remove the "this can't be right" part from the comment.

Reply:
Done, thanks for pointing out on this